setup.sh

```diff
@@ -20,12 +20,12 @@ touch ~/.sync-music/config/keys.json
 
 echo -e "\nGiving executable permissions..."
 chmod +x ./src/generate_temp_files.sh
-chmod +x ./src/sync_music.py
+chmod +x ./sync_music.py
 
 echo -e "\nCreating required symlinks..."
 if [ ! -L ~/.sync-music/scripts/generate_temp_files.sh ]; then
     ln -s ./src/generate_temp_files.sh ~/.sync-music/scripts/generate_temp_files.sh 
 fi
 if [ ! -f $HOME/bin/sync-music ]; then 
```

Comment:
One more check should be here for directory $HOME/bin (do not always available by default). If not present, then it would be better to make it before creating a link.